cypress/utils/testing-data-utils.js

```diff
@@ -252,3 +327,315 @@ export const createOrganization = (name, userEmail) => {
     workPhoneNumber: "2123892839",
   });
 };
+
+export const addPatient = ({
+  facilityId: facilityId,
```

Comment:
style question: is there a reason why this function takes the spread of variables on the `patient` object instead of just the `patient` object like what's done in `addDeviceToFacility`? Doing it this way seems redundant since we have to define both the contract here and in the caller as well as seeming to be stylistically different than the way we're doing it for facility (which I think is preferable).

If we did 

```js
# function
const addPatient = (patient) => {
    cy.makePOSTRequest({
    operationName: "AddPatient",
    variables: {
          facilityId: patient.facilityId
          firstName: patient.firstName
          ....
    }

}
```
the caller could just invoke `addPatient(patient)` rather than having to know about the shape of the patient object like in 104 of the setup utils file